Kickstarter-iOS/Views/Cells/ThanksCategoryCell.swift

```diff
@@ -5,16 +5,16 @@ import UIKit
 
 internal final class ThanksCategoryCell: UICollectionViewCell, ValueCell {
 
-  @IBOutlet fileprivate weak var bgView: UIView!
-  @IBOutlet fileprivate weak var exploreLabel: UILabel!
-  @IBOutlet fileprivate weak var liveProjectCountLabel: UILabel!
+  @IBOutlet private weak var bgView: GradientView!
+  @IBOutlet private weak var exploreLabel: UILabel!
+  @IBOutlet private weak var liveProjectCountLabel: UILabel!
 
   func configureWith(value category: KsApi.Category) {
-    _ = self.bgView
-      |> UIView.lens.backgroundColor .~ (UIColorFromCategoryId(category.id) ?? .ksr_text_navy_900)
+    let (startColor, endColor) = discoveryGradientColors(forCategoryId: category.root?.id)
+    self.bgView.setGradient([(startColor, 0.0), (endColor, 1.0)])
```

Comment:
nice! thanks for doing this treatment on the thanks screen. i have a trello card somewhere i can archive now